src/solver.f90

```diff
@@ -37,23 +38,39 @@ module m_solver
       !! for later use.
 
       real(dp) :: dt, nu
+      integer :: n_iters, n_output
```

Comment:
Are these different pr the same as the values in `src/common.f90`?

Reply:
They're the same, but we need them here as well because `globs_t` is not a member of the solver class yet. Input parameters right now handled very poorly, we'll need to work on them when we're adding the capability to read input parameters from an input file in a later PR.

Reply:
ah I see